packages/generator-helper/src/dmmf.ts

```diff
@@ -116,6 +115,10 @@ export namespace DMMF {
       model?: SchemaEnum[]
       prisma: SchemaEnum[]
     }
+    fieldRefTypes: {
```

Comment:
It could have been easier to understand perhaps if it was called inputRefTypes (I think you suggested this @weakky)